terraform/environments/corporate-staff-rostering/locals_security_groups.tf

```diff
@@ -546,6 +558,470 @@ locals {
+        }
+        ldap_tcp_domain = {
+          description = "389: TCP Allow LDAP ingress from Azure DC"
+          from_port   = 389
+          to_port     = 389
+          protocol    = "TCP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+          # NOTE: not completely clear this is needed as it's not in the existing Azure SG's
+        }
+        ldap_udp_domain = {
+          description = "389: UDP Allow LDAP ingress from Azure DC"
+          from_port   = 389
+          to_port     = 389
+          protocol    = "UDP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+          # NOTE: not completely clear this is needed as it's not in the existing Azure SG's
+        }
+
+        smb_tcp_domain = {
+          description = "445: TCP SMB ingress from Azure DC"
+          from_port   = 445
+          to_port     = 445
+          protocol    = "TCP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+        smb_udp_domain = {
+          description = "445: UDP SMB ingress from Azure DC"
+          from_port   = 445
+          to_port     = 445
+          protocol    = "UDP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+        rpc_dynamic_udp_domain = {
+          description = "49152-65535: UDP Dynamic Port range"
+          from_port   = 49152
+          to_port     = 65535
+          protocol    = "UDP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+        rpc_dynamic_tcp_domain = {
+          description = "49152-65535: TCP Dynamic Port range"
+          from_port   = 49152
+          to_port     = 65535
+          protocol    = "TCP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+      }
+      egress = {
+        all = {
+          description = "Allow all traffic outbound"
+          from_port   = 0
+          to_port     = 0
+          protocol    = "-1"
+          cidr_blocks = ["0.0.0.0/0"]
+        }
+      }
+    }
+    database = {
+      description = "New security group for database servers"
+      ingress = {
+        all-from-self = {
+          description = "Allow all ingress to self"
+          from_port   = 0
+          to_port     = 0
+          protocol    = -1
+          self        = true
+        }
+        # IMPORTANT: check if an 'allow all from load-balancer' rule is required
+        echo_core_tcp_db = {
+          description = "7: Allow ingress from port 7 oem agent echo" # Not sure what this is
+          from_port   = 7
+          to_port     = 7
+          protocol    = "TCP"
+          cidr_blocks = local.security_group_cidrs.core
```

Comment:
Use local.security_group_cidrs.oracle_oem_agent.  This needs to include hmpps-oem data subnet (maybe we just allow the VPC).  And it's fine to include the Azure core CIDR as well

Reply:
azure-core CIDR included
think I've got the VPC in here already, wasn't sure where/how to reference the hmpps-oem data subnet you mentioned

Reply:
VPC included